share/instruments/instruments.xml

```diff
@@ -213,6 +213,24 @@
                         <program value="73"/>
                   </Channel>
             </Instrument>
+            <Instrument id="baroque-flute">
+                  <longName>Baroque Flute</longName>
+                  <shortName>Bq. Fl.</shortName>
+                  <description>Baroque Travers Flute</description>
```

Comment:
Is that Information used anywhere? I see it more like a comment. Or are you complaining about a missing e, "Traverse"?